backend/python/backend_outlines/run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+##
+## A bash script wrapper that runs the outlines server with conda
+export PATH=$PATH:/opt/conda/bin
+
+# Activate conda environment
+source activate outlines
+
+# get the directory where the bash script is located
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+python $DIR/backend_outlines.py $@
```

Comment:
make sure the file has executable bits, and the wrapper needs to be also listed here:

https://github.com/mudler/LocalAI/blob/991ecce00462db955934ec1293954b7d61c58ad2/Dockerfile#L15

Reply:
Done